end-to-end-tests/src/main/java/com/amazonaws/blox/integ/DataServiceSupport.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.integ;
+
+import com.amazonaws.blox.dataservicemodel.v1.client.DataService;
+import com.amazonaws.blox.dataservicemodel.v1.serialization.DataServiceMapperFactory;
+import com.amazonaws.blox.integ.CloudFormationStacks.CfnStack;
+import com.amazonaws.blox.jsonrpc.JsonRpcLambdaClient;
+import lombok.RequiredArgsConstructor;
+import software.amazon.awssdk.services.lambda.LambdaAsyncClient;
+
+@RequiredArgsConstructor
+public class DataServiceSupport {
```

Comment:
instead of having a class for each service should we have something that just sets up the entire service stack?

Reply:
As mentioned, this doesn't do the stack setup, it's just pulling values out of cloudformation.